Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -223,6 +232,32 @@ public class PBXProjGenerator {
             return Set(files.map { $0.buildFile.reference })
         }
 
+        func getRunScript(runScript: RunScript) throws -> PBXShellScriptBuildPhase {
+
+            let shellScript: String
+            switch runScript.script {
+            case let .path(path):
+                shellScript = try getPath(path).read()
+            case let .script(script):
+                shellScript = script
+            }
+            let escapedScript = shellScript.replacingOccurrences(of: "\"", with: "\\\"").replacingOccurrences(of: "\n", with: "\\n")
+            let shellScriptPhase = PBXShellScriptBuildPhase(
+                reference: generateUUID(PBXShellScriptBuildPhase.self, String(describing: runScript.name) + shellScript),
+                files: [],
+                name: runScript.name ?? "Run Script",
+                inputPaths: Set(runScript.inputFiles),
+                outputPaths: Set(runScript.outputFiles),
+                shellPath: runScript.shell ?? "/bin/sh",
```

Comment:
What about making `shell` non-optional and using a default value in the `RunScript` constructor with `/bin/sh`. Since the shell is required for creating a `PBXShellScriptBuildPhase` I'd make it non-optional here as well. What do you think?